### ISSUE TITLE
after updating packages and refreshing the list, this error came up.

### ISSUE THREAD

**Issue by Borreltje** [os-js/OS.js#459]
An unexpected error occured, maybe a bug.

> Please report this if you think this is a bug. Include a brief description on how the error occured, and if you can; how to replicate it

```
http://192.168.75.147/packages/target/ArduinoPackageManager/combined.js?47469:295:35
http://192.168.75.147/osjs.js:1689:16
onsuccess@http://192.168.75.147/osjs.js:8867:24
onReadyStateChange@http://192.168.75.147/osjs.js:1199:25
```

**Comment by andersevenrud** [comment]
This was fixed in 502ba745c9717e662fe82c4a451e6286dd8bb3ec and will be forwarded to the Arduino team ASAP.

Thanks for reporting!

**Comment by andersevenrud** [comment]
PR: https://github.com/arduino-org/Arduino-OS/pull/23
